what_is.md

```diff
@@ -330,10 +330,14 @@ genomes (i.e. that ancestral node is not part of the tree at that position in th
 ## A framework for efficient computation
 
 
-```{todo}
-Introduction: algorithms on trees are known to be efficient (phylogenetics). We
-extend these to multiple correlated trees. Mention "dynamic programming" in passing.
-```
+Using tree structures is a common way to implement efficient computer algorithms, and
+many [phylogenetic](https://en.wikipedia.org/wiki/Phylogenetics) methods use the
+structure provided by the evolutionary tree to implement efficient so-called
+"dynamic programming" algorithms. The tree sequence structure allows these approaches
```

Comment:
```suggestion
structure provided by the evolutionary tree to implement efficient 
[dynamic programming](https://en.wikipedia.org/wiki/Dynamic_programming)
algorithms. The tree sequence structure allows these approaches
```

Reply:
Yep - a link solves it, I think.

FWIW I suspect that well over 50% of people reading this page won't have heard of "dynamic programming".